scripts/upgrade/hoodi/DeployDGLaunchHoodi.sol

```diff
@@ -0,0 +1,103 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.26;
+
+/* solhint-disable no-console */
+
+import {DGDeployArtifactLoader} from "scripts/utils/DGDeployArtifactLoader.sol";
+import {DGSetupDeployArtifacts} from "scripts/utils/contracts-deployment.sol";
+
+import {DGLaunchRolesValidatorHoodi} from "./DGLaunchRolesValidatorHoodi.sol";
+import {DGLaunchOmnibusHoodi} from "./DGLaunchOmnibusHoodi.sol";
+import {DGLaunchVerifier} from "../DGLaunchVerifier.sol";
+
+import {Duration} from "contracts/types/Duration.sol";
+import {Timestamp} from "contracts/types/Timestamp.sol";
+
+import {TimeConstraints} from "../TimeConstraints.sol";
+
+import {console} from "forge-std/console.sol";
+
+contract DeployUpgradeHoodi is DGDeployArtifactLoader {
+    function run() public {
+        address deployer = msg.sender;
+        vm.label(deployer, "DEPLOYER");
+
+        DGSetupDeployArtifacts.Context memory _deployArtifact = _loadEnv();
+
+        address timelock = address(_deployArtifact.deployedContracts.timelock);
+        address dualGovernance = address(_deployArtifact.deployedContracts.dualGovernance);
+        address emergencyGovernance = address(_deployArtifact.deployedContracts.emergencyGovernance);
```

Comment:
To simplify artifact files storing dao emergency governance now places as separate value in artifact file like: 

[dg_launch]
dao_emergency_governance = "...."

and could be read like 
``` 
import {DGLaunchConfig} from "scripts/utils/contracts-deployment.sol";
....
DGLaunchConfig.Context memory dgLaunchConfig = DGSetupDeployArtifacts.loadDGLaunchConfig(deployArtifactFileName);
dgLaunchConfig.daoEmergencyGovernance;
```